source/content/guides/disaster-recovery/03-site-goes-down.md

```diff
@@ -19,18 +19,18 @@ integration: [--]
 
 ## Open a Support Ticket
 
-In cases of downtime or significant functional failure in the Live environment, the first step is to open a support ticket. Even if you escalate the incident to your dedicated Customer Success Manager (included for all Enterprise contract customers), our support engineers will be the ones diagnosing the cause of downtime and working to get your site back up, and a ticket is the fastest way to get them up to speed and engaged.
+In cases of downtime or significant functional failure in the live environment, the first step is to open a support ticket. Even if you escalate the incident to your dedicated Customer Success Manager (available to all Enterprise contract customers), our engineering team will be the ones diagnosing the cause of the downtime and working to restore your site. Opening a ticket is the fastest way to get them up to speed and engaged.
```

Comment:
@rachelwhitton just for my own knowledge, do we keep "Live" as capitalized here since it's the proper name of an environment? I'm updating some internal docs and figured I'd just check since this is an example of it.

Reply:
"Live" should not in lowercase. It's not considered a proper noun or product name.

Reply:
Another location here, we can't say "our engineering team", because it will cause confusion. We need to say "support engineers" or "support team".